docs/source/releases/v1_12_0a0.rst

```diff
@@ -0,0 +1,16 @@
+ | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+ | # # #
+ | # # # Author:
+ | # # # Naval Research Laboratory, Marine Meteorology Division
+ | # # #
+ | # # # This program is free software: you can redistribute it and/or modify it under
+ | # # # the terms of the NRLMMD License included with this program. This program is
+ | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+ | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+ | # # # for more details. If you did not receive the license, for more information see:
+ | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Version 1.12.0a0 (2024-02-12)
+*****************************
+
+No updates.
```

Comment:
Is there a reason 1.12.0a0 and 1.13.0a0 are both added here with "No updates" ?

Reply:
I put 1.13.0a0 back to its original contents. Can we just delete 1.12.0a0?

Reply:
Yup! I think this branch was made before any updates happened in v1_13_0a0. Not sure about the no updates part. I thought that was from you but maybe it was from me. Can't remember. I'll remove it nonetheless.